packages/hub/src/lib/parse-safetensors-metadata.ts

```diff
@@ -139,12 +140,13 @@ async function parseShardedIndex(
 		throw new SafetensorParseError(`Failed to parse file ${path}: not a valid JSON.`);
 	}
 
+	const pathPrefix = path.match(RE_FILE_DIR)?.[1] ?? "";
```

Comment:
maybe clearer intent to do something like (untested code):
```ts
const parentFolder = path.substr(0, path.lastIndexOf("/"))
```
?

Reply:
handled in 75ce40a190cdc910d9ca667d0650d1d1860152bc